src/js-api-specification.md

```diff
@@ -0,0 +1,145 @@
+| PRESCRIPTIONS           | Prescriptions Hub Page                                                                                                           |
+| HEALTH_RECORDS          | Health Record Hub Page                                                                                                           |
+| SYMPTOMS                | Symptoms Hub Page                                                                                                           |
+| MESSAGES                | Messages Hub Page                                                                                                           |
+
+
+##### Example Usage
+
+```
+nhsapp.navigation.goToPage(nhsapp.navigation.AppPage.MESSAGES)
+```
+
+##### Status
+
+Live
+
+---
+
+### Storage <a name="storage"></a>
+
+#### addEventToCalendar Function <a name="addToCalendar"></a>
+
+```	javascript
+nhsapp.storage.addEventToCalendar(
+  subject: string,
+  body: string,
+  location: string,
+  startTimeEpochInSeconds: integer,
+  endTimeEpochInSeconds: integer,
+)
+```
+
+##### Description
+
+Add an event to the calendar on the user's device. The NHS App's native controls will alert the user of any issues adding the event to their calendar.
+
+#### Parameters
+
+| Parameters              |                                                                                                                                                                |
+|-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| subject                 | **Type:** string *Required*<br>The Subject line for the calendar appointment.                                                                                                            |
+| body                    | **Type:** string *Required*<br>Plain text body for the appointment.                                                                                                                      |
+| location                | **Type:** string *Required*<br>Plain text location for the appointment.                                                                                                                  |
+| startTimeEpochInSeconds | **Type:** number *Required*<br>Start datetime for the appointment, set as a Epoch timestamp.                                                                                             |
+| endTimeEpochInSeconds   | **Type:** number *Optional*<br>End datetime for the appointment, set as a Epoch timestamp. Must be greater or equal to `startTimeEpochInSeconds`.  Defaults to the same value as the start time (matching default behaviour of iOS and Android). |
+
+##### Example Usage
+
+```
+nhsapp.storage.addEventToCalendar(
+	"Hospital Appointment",
+	"Please show up,
+	"Dr Surgery, LS1",
+	1592211600,
+	1592215200
+)
```

Comment:
```suggestion
nhsapp.storage.addEventToCalendar(
	"GP Appointment",
	"GP appointment details here",
	"GP Surgery, LS1",
	1592211600,
	1592215200
)
```

Reply:
Purposely showing an off the cuff hospital example here. Secondary care is what we want to encourage this functionality for as we off GP appointments within the app and ability to add to calendar (soon)